lib/services/ibm_devops.rb

```diff
@@ -1,8 +1,9 @@
 class Service::IBMDevOpsServices < Service::HttpPost
-  string :ibm_id
+  title 'IBM Bluemix DevOps Services'
+  string :ibm_login
```

Comment:
The removal of `ibm_id` will break all existing installations of this service. Could you instead of fallback to `ibm_id` via `data["ibm_id"]` if it's present and add documentation prompting users to move to the new `ibm_login` config variable?

If you agree with this, definitely add a few tests making sure nothing breaks.